src/main.rs

```diff
@@ -188,6 +189,24 @@ struct GolemCommand {
 fn main() -> Result<(), Box<dyn std::error::Error>> {
     let command = GolemCommand::parse();
 
+    if let Some(level) = command.verbosity.log_level() {
+        let tracing_level = match level {
+            Level::Error => tracing::Level::ERROR,
+            Level::Warn => tracing::Level::WARN,
+            Level::Info => tracing::Level::INFO,
+            Level::Debug => tracing::Level::DEBUG,
+            Level::Trace => tracing::Level::TRACE,
+        };
+
+        let subscriber = FmtSubscriber::builder()
+            .with_max_level(tracing_level)
+            .with_writer(std::io::stderr)
+            .finish();
+
+        tracing::subscriber::set_global_default(subscriber)
+            .expect("setting default subscriber failed");
```

Comment:
if it will fail, what message will be visible by user?

`setting default subscriber failed`

?
if yes, then

```
.expect("setting verbosity level failed")
```
probably making more sense

Reply:
It's not even log - it is a crash. No point in making it user friendly - user can not fix it.

Reply:
We could use `.unwrap()` here. `expect` is a bit better for code maintenance.